lib/temporal/workflow/history/event.rb

```diff
@@ -42,8 +42,8 @@ def initialize(raw_event)
         end
 
         # Returns the ID of the first event associated with the current event,
-        # referred to as a "decision" event. Not related to DecisionTask.
-        def decision_id
+        # referred to as a "workflow_task" event. Not related to WorkflowTaskTask.
```

Comment:
^ This comment explain exactly what is wrong with the rename :)

`decision_id` should be `command_id`